ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Alpine.yml

```
@@ -20,6 +20,9 @@ Build_Tool_Packages:
   - freetype-dev
   - grep
   - gnupg
+  - gnutls
+  - gnutls-dev
+  - gnutls-utils
```

```suggestion
  - gnutls                        # OpenSSL tests
  - gnutls-dev                    # OpenSSL tests
  - gnutls-utils                  # OpenSSL tests
```
Making this suggestion in the UI with a proportional font but I think I've got the right amount of spaces here. It's good to have a note of why we've added things if they're for a specific purpose.